### PR TITLE
New package: bfg-1.13.0

### PR DESCRIPTION
**Pull request by Monkey-7-12**
BFG Repo-Cleaner
Removes large or troublesome blobs like git-filter-branch does, but faster. And written in Scala

### CODE REVIEW

File: srcpkgs/bfg/template [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+do_build() {
+	echo "#!/bin/sh" > ${pkgname}
+	echo "java -jar /usr/share/${pkgname}/${pkgname}-${version}.jar \$@" >> ${pkgname}
+	chmod 755 ${pkgname}
+}
+
+do_install() {
+	# Main class
+	vbin ${pkgname}
+	vinstall $XBPS_SRCDISTDIR/${pkgname}-${version}/${pkgname}-${version}.jar 644 usr/share/${pkgname} ${pkgname}-${version}.jar
```
Comment (pullmoll) [inline srcpkgs/bfg/template]: '<filename>` is redundant here (strip 4th parameter)

File: srcpkgs/bfg/template [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+}
+
```
Comment (pullmoll) [inline srcpkgs/bfg/template]: Remove empty lines

File: srcpkgs/bfg/template [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+depends="virtual?java-runtime"
+short_desc="BFG Repo-Cleaner"
+maintainer="Monkey 7 <monkeyseven@fblog.ch>"
+license="GPL-3.0-or-later"
+homepage="https://rtyley.github.io/bfg-repo-cleaner"
+distfiles="https://repo1.maven.org/maven2/com/madgag/bfg/${version}/bfg-${version}.jar"
+checksum=bf22bab9dd42d4682b490d6bc366afdad6c3da99f97521032d3be8ba7526c8ce
+skip_extraction="bfg-${version}.jar"
+
+
```
Comment (pullmoll) [inline srcpkgs/bfg/template]: Remove empty line.

**Comment by Monkey-7-12** [comment]
@pullmoll
thanks for the review

**Comment by pullmoll** [comment]
Please squash the commits into one.

**Pull request closed without merging**